addon/Config.jsm

```diff
@@ -82,23 +82,23 @@ var config = {
   // a place to put an 'isEligible' function
   // Will run only during first install attempt
   "isEligible": async function() {
-    // get whatever prefs, addons, telemetry, anything!
-    // Cu.import can see 'firefox things', but not package things.
-    return true;
+    // We only support this study on Windows.
+    return AppConstants.platform === "win";
   },
 
-  /* Button study branches and sample weights
-     - test kittens vs. puppies if we can only have one.
-       - downweight lizards.  Lizards is a 'poison' branch, meant to
-         help control for novelty effect
-  */
+  /**
+   * We want 20% of users to be in the control group: nothing
+   * should happen there.
+   */
   "weightedVariations": [
-    {"name": "kittens",
-      "weight": 1.5},
-    {"name": "puppers",
-      "weight": 1.5},
-    {"name": "lizard",
-      "weight": 1},  // we want more puppers in our sample
+    {
```

Comment:
@mythmon , does this mean that 20% of the profiles get to the control variation and 80% to the test variation?

Reply:
Yes, exactly. Also, in most studies, I would usually see this as weights 1 and 4, instead of 0.2 and 0.8. They don't need to sum to 1.0, and mostly just need to have the desired ratios between each other. Nothing wrong with using the fractions, but if you'd prefer you don't need to.